cmd/pipeline/deploy.go

```diff
@@ -234,8 +236,21 @@ func (pc *Command) waitUntilRunning(ctx context.Context, name string, action *ty
 	signal.Notify(stop, os.Interrupt)
 	exit := make(chan error, 1)
 
-	go func() {
+	var wg sync.WaitGroup
 
+	wg.Add(1)
+	go func(wg *sync.WaitGroup) {
+		defer wg.Done()
+		sseClient := sse.NewClient()
```

Comment:
I wouldn't instantiate the sse here. We could add an interface to the command so that we inject a `PipelineLogger` and the sse client simply satisfies that interface

Reply:
added the client the the pipelinclient